addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18153,3 +18153,25 @@ msgstr ""
 msgctxt "#38025"
 msgid "Choose information provider"
 msgstr ""
+
+#: system/settings/rbp.xml
+msgctxt "#38026"
+msgid "Decode the stereo stream from 3D files"
+msgstr ""
+
+#. Description of setting "Decode the stereo stream from 3D files" with label #38026
+#: system/settings/rbp.xml
+msgctxt "#38027"
+msgid "This option decodes frames for both eyes from video containing Multiview Video Coding (MVC) as typically found on 3D Blu-ray. Disabling may improve performance if you don't require 3D."
+msgstr ""
+
+#: system/settings/rbp.xml
+msgctxt "#38028"
+msgid "Enable Full HD HDMI modes for stereoscopic 3D"
+msgstr ""
+
+#. Description of setting "Enable Full HD HDMI modes for stereoscopic 3D" with label #38028
+#: system/settings/rbp.xml
+msgctxt "#38029"
+msgid "This option uses frame-packing to output full resolution for 3D through HDMI"
```

Comment:
Okay. It improves video quality (twice as many pixels). 
Doesn't affect CPU particularly, but may not be supported by all TVs.